### PR TITLE
`net/minecraft/client` mappings

### PR DESCRIPTION
**Pull request by IotaBread**

### CODE REVIEW

**Review by OroArmor — APPROVED** [review]
Looks good. Have you checked for conflicts with #5?

**Comment by IotaBread** [comment]
Just fixed them

**Review by NoahvdAa — APPROVED** [review]

File: mappings/net/minecraft/client/MinecraftClient.mapping [hunk 1]
```diff
@@ -272,6 +306,10 @@ CLASS net/minecraft/unmapped/C_ayfeobid net/minecraft/client/MinecraftClient
 		ARG 4 metadata
 		ARG 5 insertionPosition
 		ARG 6 source
+	METHOD m_jtntceqm (Lnet/minecraft/unmapped/C_sddaxwyk;)Ljava/util/stream/Stream;
+		ARG 0 itemStack
+	METHOD m_jytpvdsa countryEqualsISO3 (Ljava/lang/Object;)Z
+		ARG 0 object
```
Comment (triphora) [inline mappings/net/minecraft/client/MinecraftClient.mapping]:
```suggestion
		ARG 0 o
```
Does this rule even count if it's not overriding Object.equals?

File: mappings/net/minecraft/client/option/GameOptions.mapping [hunk 2]
```diff
@@ -18,11 +18,13 @@ CLASS net/minecraft/unmapped/C_lscptbhn net/minecraft/client/option/GameOptions
 	FIELD f_cpeiozpi textBackgroundOpacity D
 	FIELD f_cxkxaksc smoothLighting Lnet/minecraft/unmapped/C_pgtmqhfp;
 	FIELD f_daisjzsp heldItemTooltips Z
+	FIELD f_dcnckdal REALLY_FAR_RENDER_DISTANCE I
```
Comment (triphora) [inline mappings/net/minecraft/client/option/GameOptions.mapping]:
```suggestion
	FIELD f_dcnckdal VERY_FAR_RENDER_DISTANCE I
```

**Review by triphora — APPROVED** [review]

**Pull request merged**